libcontainerd/client_linux.go

```diff
@@ -445,12 +447,43 @@ func (clnt *client) restore(cont *containerd.Container, options ...CreateOption)
 }
 
 func (clnt *client) Restore(containerID string, options ...CreateOption) error {
+	if clnt.liveRestore {
```

Comment:
you are reading it wrong, it was always like that, i just added the original code back from my previous PR because of backwards compat concerns